src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+
+public class Deadline extends Task{
+    public void setBy(String by) {
+        this.by = by;
+    }
+
+    public String getBy() {
+        return by;
+    }
+
+    protected String by;
```

Comment:
Variables like this should be declared at the start of the class, before any method(s) are declared :)